### DISCUSSION TITLE
Re-enabled Windows 10 classic taskbar

### DISCUSSION THREAD

**Discussion by radixliner** [valinet/ExplorerPatcher#197] [Q&A] [answered]
There is the only way to re-enable back the classic taskbar? because i was an accident disabled the classic taskbar feature...

i did try to uninstall but i didn't find the ExplorerPatcher directory on the AppData folder for some reason.

**Comment by arthuridea** [comment, marked as answer]
I have the same issue until I find settings in registry 'HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher'. You can change the  'OldTaskbar'(DWORD) value to 1,and the classic style will be back.
